src/helpers/helpers.options.js

```diff
@@ -65,6 +65,22 @@ module.exports = {
 		};
 	},
 
+	parseFontOptions: function(options, globalDefaults) {
```

Comment:
I was planning for a long time to have this helper method (see [this implementation](/chartjs/chartjs-plugin-datalabels/blob/master/src/utils.js#L83)). Can you use the same code? I would call it `helpers.options.toFont` to be consistent with `toPadding` (and because `Options` is redundant with the namespace), and I would not expose the second parameter but instead require `core.defaults` in this file and use `defaults.global`, which would simplify lot of calls to this method.

Reply:
Can you also bring these [unit tests](/chartjs/chartjs-plugin-datalabels/blob/master/test/specs/utils.spec.js#L65)

Reply:
I forgot that `font.*` options was only in the datalabels plugin and you right, it shouldn't be implemented as part of this PR. In this case, I would make this method private, `_toFont` (shorter name), until we decide of the final behavior.

I would still import unit tests (adapted)